test/GithubApi.Authentication.test.js

```diff
@@ -0,0 +1,31 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'AgileTestingColombia';
```

Comment:
please change this to be your user name `'edopore'`

Reply:
I changed username, but test don't pass anyway